database/models.py

```diff
@@ -43,7 +51,7 @@ class Quote(db.Document):
     updatedOn = db.DateTimeField(default=datetime.datetime.utcnow)
     meta = {"collection": "quotes"}
 
-    def to_json(self):
+    def to_json(self, *args, **kwargs):
```

Comment:
Do we make use of *args, **kwargs here? I don't think so. If not, then remove these.

Reply:
Actually, the pylint shows warning if I don't include them, Because we are overwriting the method it says arguments not match.

Reply:
Okay!